packages/iTwinUI-react/src/core/utils/components/VirtualScroll.tsx

```diff
@@ -389,14 +389,13 @@ export const useVirtualization = (props: VirtualScrollProps) => {
   return {
     outerProps: {
       style: {
-        overflow: 'hidden',
```

Comment:
I'm not sure about this change.

@veekeys Do you remember why this was added?

Reply:
I was trying to find the case, but could not.. So I am not sure anymore :/